packages/accordions/src/styled/accordion/StyledInnerPanel.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.step_inner_panel';
+
+export interface IStyledInnerPanel {
+  isExpanded?: boolean;
+}
+
+export const StyledInnerPanel = styled.div.attrs<IStyledInnerPanel>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledInnerPanel>`
+  transition: max-height 0.25s ease-in-out;
+  overflow: hidden;
+  max-height: ${props => !props.isExpanded && '0 !important'}; /* stylelint-disable-line */
```

Comment:
Please use a construct like this to disable stylelint and document why the `!important` is necessary: https://github.com/zendeskgarden/react-components/blob/348329cfed710b389be93ce5aa0d26f5edb59d5a/packages/dropdowns/src/styled/menu/StyledMenu.ts#L31-L32